.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,6 +44,8 @@ body:
         - 'NovaCustom NS5x 12th Gen'
         - 'NovaCustom NS7x 11th Gen'
         - 'NovaCustom NS7x 12th Gen'
+        - 'NovaCustom V54 Series'
+        - 'NovaCustom V56 Series'
```

Comment:
@mkopec shouldn't it be 'NovaCustom V56 14th Gen' for consistency?

Reply:
okay, changed in 61295a19308b0d62d6458e27c3b2adb96877e2f3